src/terraform/dss-ci-cd/dss/modules/mosquitto/main.tf

```diff
@@ -0,0 +1,52 @@
+resource "terraform_data" "image" {
+  # To prevent out-of-order deletion via provisioner command.
+  depends_on = [var.repository_created]
+  input      = "${var.repository_id}/mosquitto" # Image ID.
+  provisioner "local-exec" {
+    command = join(" ", [
+      "gcloud builds submit econia",
+      "--config cloudbuild.yaml",
+      "--substitutions",
+      join(",", [
+        "_DOCKERFILE=mqtt/Dockerfile.mosquitto",
+        "_IMAGE_ID=${self.input}"
+      ])
+    ])
+  }
+  provisioner "local-exec" {
+    when = destroy
+    command = join("\n", [
+      "result=$(gcloud artifacts docker images list --filter IMAGE=${self.output})",
+      "if [ -n \"$result\" ]; then",
+      "gcloud artifacts docker images delete ${self.output} --quiet",
+      "fi"
+    ])
+  }
+}
+
+# https://github.com/hashicorp/terraform-provider-google/issues/5832
+resource "terraform_data" "instance" {
```

Comment:
As discussed offline this should be a cloud run instance, see PostgREST as a reference

Reply:
Noting we switched to compute engine due to cloud run MQTT issues